### ISSUE TITLE
DS - Dash backups not capped

### ISSUE THREAD

**Issue by Romainrob** [SHWotever/SimHub#1243]
**Describe the bug**
Hi,
It seems the dash backup system is no longer cleaning old backups when saving resulting in an overgrowing dash template folder.

**Simhub version**
8.2.3

Bonne soirée

**Comment by SHWotever** [comment]
Hello !

I checked and the backups were cleaned as intended, on dashstudio it's not limited in number like other files but in days (mostly because if you have a ctrl+s OCD like me it would go too fast) , the files older than 5 days (at backup time) will be cleaned at the next save. The delay is evaluated based on the file creation time which is not the one shown in windows explorer by default but is set during backup to the date of the backup. so file has date likle this : 
- last modification date = original version date 
- Creation date = backup date.

![image](https://user-images.githubusercontent.com/2207331/222907723-60cd9c3f-f9b0-495a-a258-0ad450b7e253.png)


However in a way it lacking an automatic cleanup of the old backup files if you don't save the file again, I added it at simhub startup in order to remove all the unused backups after that delay.
